uaclient/snap.py

```diff
@@ -16,7 +18,7 @@ def is_installed() -> bool:
 def configure_snap_proxy(
     http_proxy: Optional[str],
     https_proxy: Optional[str],
```

Comment:
```suggestion
    http_proxy: Optional[str] = None,
    https_proxy: Optional[str] = None,
```

Reply:
Thanks for directing me here. Yes I had botched the merge conflicts and dropped that changeset in the merge.